test/init.js

```diff
@@ -231,22 +228,4 @@ describe('#init', () => {
     statsd = createHotShotsClient(['host', 1234, 'prefix', 'suffix', false, false, true], clientType);
     assert.ok(statsd.mock);
   });
-
-  it('should create a socket variable that is an instance of dgram.Socket', () => {
-    statsd = createHotShotsClient({}, clientType);
-    assert.ok(statsd.socket instanceof dgram.Socket);
-    skipClose = true;
-  });
```

Comment:
Is there a reason this was removed?  In the past people did access this directly, for better or worse, to catch errors, so this would become a backwards-compatibility issue to lose.

Reply:
Well, I was unsure what to do since `socket` is not an instance of the net/dgram anymore.